File: Unidad 2-Estructuras de Control/Ejercicio21.java
//Equipo 3
//Diego De Gante
//Programa que dado un número entero n determina su factorial.
//Entrada:num (numero deseado por el usuario ha sacar factorial)
//Salida:factor (resultado de la factorial del numero dado)

import java.util.Scanner;

public class Main{
 public static void main(String arg[]){
   
   //Definir variables
   Scanner entrada = new Scanner(System.in); 
   int num = 0, i = 0, factorial = 1;

   System.out.println("Ingrese el numero a sacar factorial");
   num = entrada.nextInt();
   
   //Ciclo for de multiplicacion
   while(num != 0){
     factorial *= num;
     num --;
   }

   System.out.println(factorial);

 }
}

File: Unidad 2-Estructuras de Control/Ejercicio21.c
//Equipo 3
//Diego De Gante
//Programa que dado un número entero n determina su factorial.
//Entrada:num (numero deseado por el usuario ha sacar factorial)
//Salida:factorial (resultado de la factorial del numero dado)

#include <stdio.h>

int main()
{
    int num = 0, factor = 1;
    printf("Ingresa el número a sacar factorial\n");
    scanf("%i", &num);

    while(num != 0){
     factor *= num;
     num --;
   }
   printf("%i\n", factor);
   return 0;
}